pages/index.js

```diff
@@ -1,16 +1,29 @@
 import styled from "styled-components";
+import useSWR from "swr";
 
-import items from "@/db.json";
 import Item from "@/components/Item";
 
 export default function HomePage() {
+  const { data: items, isLoading, error } = useSWR("/api/items");
+
+  if (error) {
+    console.log(error.message);
+    return <h2>{error.message}</h2>;
+  }
+
+  console.log(items);
+
   return (
     <StyledList>
-      {items.map((item) => (
-        <li key={item.itemId}>
-          <Item title={item.title} initialStatus={item.initiallyLost} />
-        </li>
-      ))}
+      {isLoading ? (
+        <h2>Loading...</h2>
```

Comment:
A `h2` element inside of an `ul` is not correct html [(see caninclude)](https://caninclude.glitch.me/caninclude?child=h2&parent=ul). Please find another solution. :)